messaginginapp/src/main/java/io/customer/messaginginapp/ui/InlineInAppMessageView.kt

```diff
@@ -34,24 +36,34 @@ class InlineInAppMessageView @JvmOverloads constructor(
     attrs: AttributeSet? = null,
     @AttrRes defStyleAttr: Int = 0,
     @StyleRes defStyleRes: Int = 0
-) : InlineInAppMessageBaseView(context, attrs, defStyleAttr, defStyleRes),
```

Comment:
With the addition of controller, there's likely very little shared code between XML and Compose based views. So I removed the base view for simplicity. Once we start working on Compose, we can revisit this and decide what structure works best for both view types.